content/rc/subscriptions/_index.md

```diff
@@ -22,13 +22,17 @@ This page helps you manage your Redis Cloud subscriptions; it briefly compares a
 
 As of February 2021, Redis Enterprise Cloud supports the following subscription plans:
 
-- _Free plans_ are designed for training purposes and prototyping.  They can be seamlessly upgraded to Fixed plans with no data loss.  (Free plans are a tier of Fixed plans.)
+### Free plans
```

Comment:
I found this was better because it added some menu items to the right. I can remove if needed

Reply:
I don't think the menu items are necessary, and splitting these up into sections makes it seem like the table below is a part of the Annual Plan section. I will say that instead of highlighting each plan type like before the change, it would be better to bold the plan types in the list to make them easier to read.

Reply:
If you really want to have a section for each plan type, you would need to rearrange the section entirely. I have an idea as to how you could do it, but I'm having trouble explaining it in text form, so I'll add a link here to a branch off of this one with the changes that could work once I move stuff around.

Reply:
We can revert it is you want, no worries